### PR TITLE
Add a note about minumum Go version

### PR DESCRIPTION
**Pull request by CsatariGergely**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,15 @@ In R2 release, a Hugo / Docsy based documentation site was introduced for the Ne
 
 ### Setting up the environment
 
+1. The site is using Hugo as the documentation generating engine and Hugo depends on Go in mudule handling you need to
```
Comment (liamfallon) [inline README.md:16]:
```suggestion
1. The site is using Hugo as the documentation generating engine and as Hugo depends on Go in mudule handling you need to
```

File: README.md [hunk 2]
```diff
@@ -13,6 +13,15 @@ In R2 release, a Hugo / Docsy based documentation site was introduced for the Ne
...
+   have Go installed on your computer. Most Linux distributions have Go preinstalled, but some of them have a too old
```
Comment (liamfallon) [inline README.md:17]:
```suggestion
   have Go installed on your computer. Most Linux distributions have Go preinstalled, but some of them have a verstion that is too old
```

File: README.md [hunk 3]
```diff
@@ -13,6 +13,15 @@ In R2 release, a Hugo / Docsy based documentation site was introduced for the Ne
...
+   version for Hugo. You should have at least version 1.18.0. You can install version 1.18 of go on an Ubuntu using the
```
Comment (liamfallon) [inline README.md:18]:
```suggestion
for Hugo. You should have at least version 1.18.0. You can install version 1.18 of go on an Ubuntu using the
```

**Comment by liamfallon** [comment]
I think this PR needs a rebase.

**Comment by CsatariGergely** [comment]
BANG! Merged main now. maybe a rebase would have been smarter in this case :) DO we have any policies for these in Nephio?

@liamfallon

**Comment by liamfallon** [comment]
> BANG! Merged main now. maybe a rebase would have been smarter in this case :) DO we have any policies for these in Nephio?
> 
> @liamfallon

I don't think there are any policies around rebasing.

**Pull request merged**